src/Solution/SolutionProviderInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\ErrorHandler\Solution;
+
+/**
+* The interface declares an adapter to render a solution for an event.
+* Basically, it renders the error message as-is, but possible could render a button with click-to-fix action that will be handled by an HTTP request (with middleware) back to server.
```

Comment:
```suggestion
* It may render the message as-is, but it could also provide a button with a click-to-fix action that will be handled by an async HTTP request (using a separate middleware).
```